cmd/dispatcher/main.go

```diff
@@ -41,14 +41,15 @@ type envConfig struct {
 	// Should failed deliveries be requeued in the RabbitMQ?
 	Requeue bool `envconfig:"REQUEUE" default:"false" required:"true"`
 
+	// Number of concurrent messages in flight
+	PrefetchCount int           `envconfig:"PREFETCH_COUNT" default:"10" required:"false"`
 	Retry         int           `envconfig:"RETRY" required:"false"`
 	BackoffPolicy string        `envconfig:"BACKOFF_POLICY" required:"false"`
 	BackoffDelay  time.Duration `envconfig:"BACKOFF_DELAY" required:"false"`
 }
 
 const (
 	defaultBackoffDelay = 50 * time.Millisecond
-	defaultPrefetch     = 1
 	defaultPrefetchSize = 0
```

Comment:
Where do you see that we force no-ack? https://www.rabbitmq.com/amqp-0-9-1-reference.html#domain.no-ack

That would imply we don't use acknowledgements, which is not true

Reply:
This is how I'm reading go-amqp and the Rabbit docs:

> When autoAck (also known as noAck) is true, the server will acknowledge
deliveries to this consumer prior to writing the delivery to the network.  When
autoAck is true, the consumer should not call Delivery.Ack.

https://github.com/knative-sandbox/eventing-rabbitmq/blob/c944a0c5a3bea768ce200c6812d44072b4bb7001/vendor/github.com/streadway/amqp/channel.go#L1017

> The prefetch-size is ignored if the no-ack option is set.

https://www.rabbitmq.com/amqp-0-9-1-reference.html#basic.qos.prefetch-size

We set `autoAck=false`: https://github.com/knative-sandbox/eventing-rabbitmq/blob/c944a0c5a3bea768ce200c6812d44072b4bb7001/pkg/dispatcher/dispatcher.go#L70

Reply:
Oh, I think this is bad documentation on rabbit's part. I think they mean 

> The prefetch-size is ignored if the no-ack option is set *to true*.

Since we set it to false, we are not using autoAck/noAck, and therefore prefetch size/prefetch count is still relevant

Reply:
@gerhard can you back me up on this interpretation?